docs/_plugins/include_example.rb

```diff
@@ -0,0 +1,70 @@
+require 'octopress-code-highlighter'
+require 'liquid'
+
+module Octopress
```

Comment:
If our implementation doesn't require octopress, we should rename it and cite it in the reference.